amazon-kinesis-client/src/test/java/software/amazon/kinesis/lifecycle/ShardConsumerSubscriberTest.java

```diff
@@ -448,6 +520,31 @@ public void cancel() {
         }
     }
 
+    private class RecordPublisherWithInitialFailureSubscription extends TestPublisher {
+        private int subscriptionTryCount = 0;
+
+        @Override
+        public void subscribe(Subscriber<? super RecordsRetrieved> s) {
+            subscriber = s;
+            ++subscriptionTryCount;
+            s.onSubscribe(new Subscription() {
+                @Override
+                public void request(long n) {
+                    if (subscriptionTryCount == 1) {
```

Comment:
Would it be simpler to rewrite this block as just `if (subscriptionTryCount != 1)`?

Reply:
Changed to what u suggested. Agree it seems simpler to read.